src/main/java/pl/put/poznan/buildingInfo/model/Location.java

```diff
@@ -0,0 +1,55 @@
+package pl.put.poznan.buildingInfo.model;
+
+/**
+ * Represents abstract class, gathering common Building, Level and Room properties.
+ * Describes methods extended in other classes
+ */
+public abstract class Location {
+    private String id;
+    private String name;
+
+
```

Comment:
Redundant empty line.

Reply:
It would be nice to have some rule in editorconfig to autofix it.

Reply:
Editorconfig doesn't support such feature, because sometimes it may be usefull to have more blank lines => multiline strings for example.
However, it's posible to set maximum of blank lines in IntelliJ itself.
` Settings -> code style -> Java -> blank lines -> Keep maximum blank lines`